app/models/creatorship.rb

```diff
@@ -41,24 +41,26 @@ def check_invalid
     end
   end
 
+  # Make sure that the user isn't banned or suspended.
+  def check_banned
+    return unless pseud&.user&.banned || pseud&.user&.suspended
+
+    errors.add(:base, ts("%{name} cannot be listed as a co-creator.",
+                         name: pseud.byline))
+    throw :abort
+  end
```

Comment:
Just FYI, you didn't have to move this — the only thing that determines the order of the callbacks is the order of the `validate` calls above. But it's okay to leave this as is if you don't feel like moving it back.

Reply:
Ah. I'll keep that in mind for the future. If there is no issue with it, I'd like to leave it as it's been so long I'm concerned of making things worse. lol